js/views/modals/editListing/EditListing.js

```diff
@@ -221,6 +222,13 @@ export default class extends BaseModal {
     }
   }
 
+  onClickViewListingOnWeb() {
+    const slug = this.model.get('slug');
+    if (slug) {
+      openExternal(`http://openbazaar.com/store/${app.profile.id}/${slug}`);
+    }
```

Comment:
If there is no slug, should something happen here? I think it's ok for it to silently fail, which would normally not happen. Usually in places where an error would be a code problem that shouldn't occur, we put an error message in the console.

Reply:
That makes sense; I'll add an error in the console.

Reply:
Where you going to add a console error here?